internal/server/placepage/place_page.go

```diff
@@ -258,6 +258,9 @@ func fetchBtData(
 			placePageData := row.Data.(*pb.LandingPageCache)
 			if _, ok := mergedPlacePageData[place]; !ok {
 				mergedPlacePageData[place] = placePageData
+			} else {
+				mergedPlacePageData[place].Categories = append(
```

Comment:
You can use util.MergeDedupe() function in util.go

Reply:
Done.